Hexagon Update

TylerChasse opened the pull request:
Had some problems after trying to resolve conflicts on pull request #18 so I reset to a previous commit to start fresh. I also implemented a size attribute for the hexagon class so that it's easier to edit and moved the hexagon button to the bottom with the rest.

Closes #9, Closes #21

TylerChasse committed:
Added hexagon.h file and implemented hexagon functionality

TylerChasse committed:
Fixed bug where hexagons could go slightly out of bounds

TylerChasse committed:
Cleaned up hexagon.h

TylerChasse committed:
Removed radius from hexagon class

TylerChasse committed:
Merge branch 'dev' into hexagon

TylerChasse committed:
Added size to hexagon class

TylerChasse committed:
Pulled and resolved conflicts

StrawberyCrepeCookie commented:
Just a question regarding merging pull requests...
If there are lines of code in dev, for example lines 42-52 in sandbox.h on this branch, those lines are being "used" already in dev. Would merging overwrite those lines? if so, then how could I remedy that so I can complete this merge request?

TylerChasse commented:
From my understanding, as long as it says the branch has no conflicts which seems to be the case we shouldn't have to worry about anything being overwritten. For example addTriangle() in sandbox.h from lines 42-52 isn't being overwritten it's now just moved below the addHexagon() function that I added.

StrawberyCrepeCookie commented:
Alright cool, thank you for clarifying. I'll complete this request.
